### DISCUSSION TITLE
Bug Report lights in shuttle

### DISCUSSION THREAD

**Discussion by Minedeployder** [FakeFishGames/Barotrauma#13866] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

In those particular shuttles(mines) red light components are not glowing at all, no matter what im trying to press. Sometimes they fixes, i save, load and then it is not working again. Same in actual game, not even sub editor.

[The Patchwork Minelayer.zip](https://github.com/FakeFishGames/Barotrauma/files/15139695/The.Patchwork.Minelayer.zip)

https://github.com/FakeFishGames/Barotrauma/assets/70541484/6998ad93-80ed-4789-bac7-6fc340fada22

### Reproduction steps

1. Open I II or III in sub editor
2. Find light component
3. Try to light it up and fail

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.4.5.0 (Blood in the Water Update, hotfix 1)

### Which operating system did you encounter this bug on?

Windows

**Comment by Minedeployder** [comment]
btw happens with any lamps

**Comment by Regalis11** [comment]
It seems the lights are set to be hidden in-game, that will make them not emit light.

**Reply by Minedeployder** [reply]
But the lights are not turned on even when they are not hidden... I even pressed that button in video.
